Library/Homebrew/manpages/brew.1.md

```diff
@@ -15,8 +15,7 @@ didn't include with OS X.
 
 For the full command list, see the COMMANDS section.
 
-With `--verbose` or `-v`, many commands print extra debugging information.
-Note that these flags should only appear after a command.
+With `--verbose` or `-v`, many commands print extra debugging information. With `--debug` or `-d`, additional debugging output and behaviors are enabled. Details are documented under the individual commands below. Note that these flags should only appear after a command.
```

Comment:
Maybe the item for `install` could have a note about `--verbose` doing full build output, since that's the behavior for that particular subcommand?

The rest of the man page isn't one sentence per line; it's manually hard-wrapped at about 80 characters per line. I reformatted this section to be the same way.